### DISCUSSION TITLE
Which tool can be used to work as Resistor & Capacitor

### DISCUSSION THREAD

**Discussion by kaviyarasu34** [hneemann/Digital#976] [Q&A] [answered]
Hi,
First of all thank you for this awesome tool. I am a newbie to learn these awesome  things . In some areas i need potentiometer & capacitor. I just want any ic chip or some other method which can be used to perform potentiometer Something like Digital Potentiometers (digipot) & capacitor function. 

Seeking Additional help:
Sometimes i have to give ground connection from one terminal to push button, dip switch instead of power. In breadboard it is possible because we can use two terminals. Since here we have one terminal I used not gate and it solved some areas. And for  ee prom to hex decimal display inverter not working . So i changed hex values and got output.I can understand i have to program something different because changing hex values is time consuming . 
However any idea for adding ground to pushbutton/dip switch like in breadboard two terminal connection.
Guide me. 
Thank you.

**Comment by hneemann** [comment, marked as answer]
If you want to simulate resistors and capacitors, you will need an analog simulator. 
A very well known tool is LTspice. It is not open source, but it is free of charge and easy to use.

If it is only about a push button with pull up resistor, you can also build it with this simulator:

![z](https://user-images.githubusercontent.com/7190563/176595492-823f454f-677f-4fdb-a5fe-daa89393fffa.png)

**Reply by kaviyarasu34** [reply]
Thank you. i will check it out.

**Reply by kaviyarasu34** [reply]
hi, in this video i got output , after some workouts. Now i need 7 segment display to show fixed instead of blinking look.  I increased clock frequency still some other efforts needed for me. . how to do it? Guide me . even i am ready to use another set of 4 (7 segment display) to store and show the display output in fixed look.



https://user-images.githubusercontent.com/41732151/176858054-c329092c-2786-4e3f-989f-f71db1428c48.mp4

.

**Reply by hneemann** [reply]
@kaviyarasu34 
Increase the "Persistance Of Vision" value in the seven-seg settings.

**Reply by kaviyarasu34** [reply]
Thank You very much. It's working. Cool Feature!

**Comment by xotmatrix** [comment]
For analog stuff I like Iain Sharp's JavaScript port of Paul Falstad's Java Circuit simulator. It's not as full-featured as LTspice but it's easy to use and pretty helpful.

Original by Paul Falstad.
http://www.falstad.com/

JavaScript conversion by Iain Sharp.
http://lushprojects.com/

**Reply by kaviyarasu34** [reply]
Thank you. i will check it out.
